packages/design-system/members-list/index.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "./styled";
 import { Props } from "./types";
 
-export const MembersList = ({ membersImages, totalMembers }: Props) => (
+export const MembersList = ({ membersImages, totalMembers, label }: Props) => (
```

Comment:
It's easier to make `label = "Members"`
This way, if we need to use the label elsewhere, it'll work out of the box, instead of having to do `label ? label : "Members"` everywhere